<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>424</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Reflex game</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_40">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>255</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_49">
    <property name="geometry">
     <rect>
      <x>611</x>
      <y>255</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ScoreHistory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>365</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Score history</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_33">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>163</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_48">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>301</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_35">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>209</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>25</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_29">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>163</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>71</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>25</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_37">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>255</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_43">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>301</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>71</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>25</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_26">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>117</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_41">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>209</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>611</x>
      <y>25</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>25</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_47">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>301</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>611</x>
      <y>71</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_42">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>301</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_45">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>301</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_53">
    <property name="geometry">
     <rect>
      <x>611</x>
      <y>209</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_44">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>301</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_39">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>255</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_22">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>163</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>71</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>71</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>25</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_51">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>255</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>117</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>25</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>71</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_38">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>255</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_32">
    <property name="geometry">
     <rect>
      <x>611</x>
      <y>163</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_50">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>255</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_55">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>209</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_30">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>163</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_27">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>117</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>71</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_19">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>163</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_23">
    <property name="geometry">
     <rect>
      <x>611</x>
      <y>117</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="GameStatus">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>365</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start the game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_54">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>209</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_52">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>255</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_36">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>209</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_28">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>163</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_31">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>209</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_25">
    <property name="geometry">
     <rect>
      <x>439</x>
      <y>117</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>71</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_34">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>209</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_46">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>301</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>117</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_21">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>117</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>25</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_56">
    <property name="geometry">
     <rect>
      <x>611</x>
      <y>301</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_20">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>163</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_24">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>117</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>374</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Score: 0</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="prev">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>365</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>pushButton_40</zorder>
   <zorder>pushButton_49</zorder>
   <zorder>ScoreHistory</zorder>
   <zorder>pushButton_33</zorder>
   <zorder>pushButton_48</zorder>
   <zorder>pushButton_35</zorder>
   <zorder>pushButton_10</zorder>
   <zorder>pushButton_29</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_37</zorder>
   <zorder>pushButton_43</zorder>
   <zorder>pushButton_11</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_26</zorder>
   <zorder>pushButton_41</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_47</zorder>
   <zorder>pushButton_17</zorder>
   <zorder>pushButton_42</zorder>
   <zorder>pushButton_45</zorder>
   <zorder>pushButton_53</zorder>
   <zorder>pushButton_44</zorder>
   <zorder>pushButton_39</zorder>
   <zorder>pushButton_22</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_15</zorder>
   <zorder>pushButton_1</zorder>
   <zorder>pushButton_51</zorder>
   <zorder>pushButton_18</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_14</zorder>
   <zorder>pushButton_38</zorder>
   <zorder>pushButton_32</zorder>
   <zorder>pushButton_50</zorder>
   <zorder>pushButton_55</zorder>
   <zorder>pushButton_30</zorder>
   <zorder>pushButton_27</zorder>
   <zorder>pushButton_12</zorder>
   <zorder>pushButton_19</zorder>
   <zorder>pushButton_23</zorder>
   <zorder>GameStatus</zorder>
   <zorder>pushButton_54</zorder>
   <zorder>pushButton_52</zorder>
   <zorder>pushButton_36</zorder>
   <zorder>pushButton_28</zorder>
   <zorder>pushButton_31</zorder>
   <zorder>pushButton_25</zorder>
   <zorder>pushButton_13</zorder>
   <zorder>pushButton_34</zorder>
   <zorder>pushButton_46</zorder>
   <zorder>pushButton_16</zorder>
   <zorder>pushButton_21</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_56</zorder>
   <zorder>pushButton_20</zorder>
   <zorder>pushButton_24</zorder>
   <zorder>label</zorder>
   <zorder>prev</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
